### PR TITLE
tiup version info modify(v4.0 -> v5.0)

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Necessary updates for TiUP-related documents for release 5.0.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/5187

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: production-deployment-using-tiup.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ TiUP 支持管理多个 TiDB 集群，该命令会输出当前通过 TiUP cluste
 Starting /home/tidb/.tiup/components/cluster/v1.0.0/cluster list
```
Comment (lucklove) [inline production-deployment-using-tiup.md:161]: Change `/home/tidb/.tiup/components/cluster/v1.0.0/cluster list` to `/home/tidb/.tiup/components/cluster/v1.3.0/cluster list`

**Review by lucklove — COMMENTED** [review]
Rest LGTM

File: production-deployment-using-tiup.md (hunk 1 above)
Comment (TomShawn) [reply production-deployment-using-tiup.md:161]:
```suggestion
Starting /home/tidb/.tiup/components/cluster/v1.3.0/cluster list
```

File: production-deployment-using-tiup.md [hunk 2]
```diff
@@ -133,13 +133,13 @@ aliases: ['/docs-cn/dev/production-deployment-using-tiup/','/docs-cn/dev/how-to/
...
 ```shell
-tiup cluster deploy tidb-test v4.0.0 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
+tiup cluster deploy tidb-test v5.0.0 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
 ```
 
 以上部署命令中：
 
 - 通过 TiUP cluster 部署的集群名称为 `tidb-test`
-- 部署版本为 `v4.0.0`，最新版本可以通过执行 `tiup list tidb` 来查看 TiUP 支持的版本
+- 部署版本为 `v5.0.0`，最新版本可以通过执行 `tiup list tidb` 来查看 TiUP 支持的版本
```
Comment (kissmydb) [inline production-deployment-using-tiup.md:142]:
```suggestion
- 最新可用版本可以通过执行 `tiup list tidb` 来查看 TiUP 支持的版本，后续内容以版本 `v5.0.0` 为例
```

**Review by kissmydb — COMMENTED** [review]
Rest LGTM

File: production-deployment-using-tiup.md [hunk 3]
```diff
@@ -137,13 +137,13 @@ aliases: ['/docs-cn/dev/production-deployment-using-tiup/','/docs-cn/dev/how-to/
...
 ```shell
-tiup cluster deploy tidb-test v4.0.0 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
+tiup cluster deploy tidb-test v5.0.0 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
 ```
 
 以上部署命令中：
 
 - 通过 TiUP cluster 部署的集群名称为 `tidb-test`
-- 部署版本为 `v4.0.0`，最新版本可以通过执行 `tiup list tidb` 来查看 TiUP 支持的版本
+- 最新可用版本可以通过执行 `tiup list tidb` 来查看 TiUP 支持的版本，后续内容以版本 `v5.0.0` 为例
```
Comment (yikeke) [inline production-deployment-using-tiup.md:146]:
```suggestion
- 可以通过执行 `tiup list tidb` 来查看 TiUP 支持的最新可用版本，后续内容以版本 `v5.0.0` 为例
```

**Review by yikeke — APPROVED** [review]
Rest LGTM

**Pull request merged**